### DISCUSSION TITLE
change in current

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2273] [General]
If I set a charge and discharge experiment, the battery capacity is 5Ah, the discharge rate is 1C, that is, the discharge current is 5A. As the experiment progresses, the battery capacity declines. The discharge rate is still set to 1C, whether the discharge current is reduced, or is still 5A.

**Comment by rtimms** [comment]
The parameter “Nominal cell capacity [A.h]” is always used to convert between C-rate and current.

**Reply by Abhinav-Kshirsagar** [reply]
Hello @rtimms ,

I have a follow -up question. 
Does it mean that if I specify the 'Charge Current' instead of 'C-rate', the parameter "Nominal cell capacity [A.h]" will not play any role?
I tried running experiments by selecting the multiple different "Nominal cell capacity [A.h]": values but the output results seem to be unaffected.
I am running a simple charging process starting from 25% SOC till reaching 4.2V.

Thanks in advance for your response.

**Reply by rtimms** [reply]
Yes, "Nominal cell capacity [A.h]" doesn't do anything other than convert a C-rate to a current.

**Reply by Abhinav-Kshirsagar** [reply]
Thank you for the clarification!
